api/src/main/java/org/openmrs/api/db/hibernate/HibernateAdministrationDAO.java

```diff
@@ -203,18 +211,82 @@ public void setApplicationContext(ApplicationContext applicationContext) throws
 	 */
 	@Override
 	public void validate(Object object, Errors errors) throws DAOException {
+		
```

Comment:
The code in you had in this commit 71fc81ddb092939e33f9e1f6b2e54fb8623a9b11 seems more correct to me as long you address the comments on it